packages/react-text/src/typographyStyles/typographyStyles.ts

```diff
@@ -0,0 +1,72 @@
+import { MakeStylesStyleFunctionRule } from '@fluentui/make-styles';
+import { Theme } from '@fluentui/react-theme';
+
+/**
+ * Make-styles rules for the typography variants
+ */
+export const typographyStyles: {
+  caption: MakeStylesStyleFunctionRule<Theme>;
+  body: MakeStylesStyleFunctionRule<Theme>;
+  subheadline: MakeStylesStyleFunctionRule<Theme>;
+  headline: MakeStylesStyleFunctionRule<Theme>;
+  title3: MakeStylesStyleFunctionRule<Theme>;
+  title2: MakeStylesStyleFunctionRule<Theme>;
+  title1: MakeStylesStyleFunctionRule<Theme>;
+  largeTitle: MakeStylesStyleFunctionRule<Theme>;
+  display: MakeStylesStyleFunctionRule<Theme>;
+} = {
+  caption: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.base[200],
+    lineHeight: theme.global.type.lineHeights.base[200],
+    fontWeight: theme.global.type.fontWeights.regular,
+  }),
+  body: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.base[300],
+    lineHeight: theme.global.type.lineHeights.base[300],
+    fontWeight: theme.global.type.fontWeights.regular,
+  }),
+  subheadline: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.base[400],
+    lineHeight: theme.global.type.lineHeights.base[400],
+    fontWeight: theme.global.type.fontWeights.semibold,
+  }),
+  headline: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.base[500],
+    lineHeight: theme.global.type.lineHeights.base[500],
+    fontWeight: theme.global.type.fontWeights.semibold,
+  }),
+  title3: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.base[600],
+    lineHeight: theme.global.type.lineHeights.base[600],
+    fontWeight: theme.global.type.fontWeights.semibold,
+  }),
+  title2: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.hero[700],
+    lineHeight: theme.global.type.lineHeights.hero[700],
+    fontWeight: theme.global.type.fontWeights.semibold,
+  }),
+  title1: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.hero[800],
+    lineHeight: theme.global.type.lineHeights.hero[800],
+    fontWeight: theme.global.type.fontWeights.semibold,
+  }),
+  largeTitle: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.hero[900],
+    lineHeight: theme.global.type.lineHeights.hero[900],
+    fontWeight: theme.global.type.fontWeights.semibold,
+  }),
+  display: theme => ({
+    fontFamily: theme.global.type.fontFamilies.base,
+    fontSize: theme.global.type.fontSizes.hero[1000],
+    lineHeight: theme.global.type.lineHeights.hero[1000],
+    fontWeight: theme.global.type.fontWeights.semibold,
+  }),
```

Comment:
We can can existing export structure, but make it tree shakable if we will use `export * as`:

- with `export {}` (current), [REPL](https://rollupjs.org/repl/?version=2.53.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMHR5cG9ncmFwaHlTdHlsZXMlMjAlN0QlMjBmcm9tJTIwJy4lMkZ0ZXh0LWluZGV4JyU1Q24lNUNuY29uc29sZS5sb2codHlwb2dyYXBoeVN0eWxlcy5jYXB0aW9uKSUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMnRleHQtaW5kZXguanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyZXhwb3J0JTIwJTdCJTIwdHlwb2dyYXBoeVN0eWxlcyUyMCU3RCUyMGZyb20lMjAnLiUyRnRleHQtc3R5bGVzLmpzJyUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJ0ZXh0LXN0eWxlcy5qcyUyMiUyQyUyMmNvZGUlMjIlM0ElMjJleHBvcnQlMjBjb25zdCUyMHR5cG9ncmFwaHlTdHlsZXMlMjAlM0QlMjAlN0IlNUNuJTVDdGNhcHRpb24lM0ElMjAoKSUyMCUzRCUzRSUyMCglN0IlMjBjb2xvciUzQSUyMCdyZWQnJTIwJTdEKSUyQyU1Q24lMjAlMjB0aXRsZSUzQSUyMCgpJTIwJTNEJTNFJTIwKCU3QiUyMGNvbG9yJTNBJTIwJ3JlZCclMjAlN0QpJTVDbiU3RCUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE)
- with `export * as`, [REPL](https://rollupjs.org/repl/?version=2.53.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmltcG9ydCUyMCU3QiUyMHR5cG9ncmFwaHlTdHlsZXMlMjAlN0QlMjBmcm9tJTIwJy4lMkZ0ZXh0LWluZGV4JyU1Q24lNUNuY29uc29sZS5sb2codHlwb2dyYXBoeVN0eWxlcy5jYXB0aW9uKSUyMiUyQyUyMmlzRW50cnklMjIlM0F0cnVlJTdEJTJDJTdCJTIybmFtZSUyMiUzQSUyMnRleHQtaW5kZXguanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyZXhwb3J0JTIwKiUyMGFzJTIwdHlwb2dyYXBoeVN0eWxlcyUyMGZyb20lMjAnLiUyRnRleHQtc3R5bGVzLmpzJyUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCUyQyU3QiUyMm5hbWUlMjIlM0ElMjJ0ZXh0LXN0eWxlcy5qcyUyMiUyQyUyMmNvZGUlMjIlM0ElMjJleHBvcnQlMjBjb25zdCUyMGNhcHRpb24lMjAlM0QlMjAoKSUyMCUzRCUzRSUyMCglN0IlMjBjb2xvciUzQSUyMCdyZWQnJTIwJTdEKSU1Q25leHBvcnQlMjBjb25zdCUyMHRpdGxlJTIwJTNEJTIwKCklMjAlM0QlM0UlMjAoJTdCJTIwY29sb3IlM0ElMjAncmVkJyUyMCU3RCklMjIlMkMlMjJpc0VudHJ5JTIyJTNBZmFsc2UlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyZXMlMjIlMkMlMjJuYW1lJTIyJTNBJTIybXlCdW5kbGUlMjIlMkMlMjJhbWQlMjIlM0ElN0IlMjJpZCUyMiUzQSUyMiUyMiU3RCUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlN0QlMkMlMjJleGFtcGxlJTIyJTNBbnVsbCU3RA==)

Reply:
Awesome suggestion! Implemented changes

Reply:
Unfortunately API extractor does not support `export * as _` syntax My PR #18896 suffers from the same problem.

microsoft/rushstack#2780